orga/tests/integration/components/password-reset-modal-test.js

```diff
@@ -1,126 +1,190 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
 import { click, render } from '@ember/test-helpers';
-import { run } from '@ember/runloop';
 import { resolve } from 'rsvp';
 import hbs from 'htmlbars-inline-precompile';
 import Service from '@ember/service';
-import Object from '@ember/object';
+import EmberObject from '@ember/object';
 import { triggerCopySuccess } from 'ember-cli-clipboard/test-support';
 import faker from 'faker';
 
 module('Integration | Component | password-reset-modal', function(hooks) {
 
   setupRenderingTest(hooks);
 
-  let username;
-  let email;
-  let student;
+  module('Student without GAR connection method', function() {
```

Comment:
Un contexte d’exécution est implémenté par module + libellé qui commence par when
Dans ce cas, le libellé du test ne reprend pas celui du contexte.

module("when student is authenticated with GAR", async function() {
    test('should XXX', async function(assert) { })
}

Reply:
Done